### ISSUE TITLE
Failed to instantiate module permission.ui

### ISSUE THREAD

**Issue by AdamGerthel** [Narzerus/angular-permission#224]
Using v3.0 I'm having trouble adding permission.ui as a dependency. Am getting the following error:

> Failed to instantiate module permission.ui

Also, is ui-router-extras required?

**Comment by masterspambot** [comment]
> Also, is ui-router-extras required?
No it's not. It only requires `ui.router` and `permission` module:
https://github.com/Narzerus/angular-permission/blob/master/src/permission-ui/permission-ui.js#L146

Did you include both of them?

**Comment by AdamGerthel** [comment]
Yes, I included both. I haven't spent much time debugging it though since I don't require the ui.router compatibility yet.

I'm using uglify to combine all my external scripts, and I've loaded permission before permission.ui: 
![screen shot 2016-04-25 at 15 51 27](https://cloud.githubusercontent.com/assets/1490370/14786069/96980c2a-0afd-11e6-9b25-8366ac314969.png)

![screen shot 2016-04-25 at 15 51 53](https://cloud.githubusercontent.com/assets/1490370/14786092/a6d24d08-0afd-11e6-96f9-abc7a650b989.png)

I'm using 3.0, installed via Bower (requested 3.0.0 specifically).
